your-delivery/src/main/java/com/ch/yourdelivery/common/handler/DeliveryExceptionHandler.java

```diff
@@ -0,0 +1,32 @@
+package com.ch.yourdelivery.common.handler;
+
+import com.ch.yourdelivery.common.exception.ExceptionCode;
+import com.ch.yourdelivery.common.exception.DeliveryException;
+import jakarta.servlet.http.HttpServletRequest;
+import jakarta.servlet.http.HttpServletResponse;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestControllerAdvice;
+
+@RestControllerAdvice
+public class DeliveryExceptionHandler {
+
+    private final String CONTEXT_TYPE = "application/json;charset=UTF-8";
+
+    @ResponseStatus(HttpStatus.CONFLICT)
+    @ExceptionHandler(value = DeliveryException.class)
+    public void conflictExceptionHandler(HttpServletResponse response, DeliveryException deliveryException){
+        response.setContentType(CONTEXT_TYPE);
+        
+        deliveryException.getExceptionCode().getMessage();//공통 response 후 사용
+        deliveryException.getExceptionCode().getHttpStatus();//공통 response 후 사용
+        
+        if(deliveryException.getExceptionCode().equals(ExceptionCode.CONFLICT)){
+            response.setStatus(HttpStatus.CONFLICT.value());
+        }else{
+            response.setStatus(HttpStatus.INTERNAL_SERVER_ERROR.value());
+        }
```

Comment:
지난번 맨토링때 말씀드렸던부분이 반영이 안된것같아요. 
```suggestion
        HttpStatus status = deliveryException.getExceptionCode().getHttpStatus();
        response.setStatus(status);
```

혹시 위쪽에서 `//공통 response 후 사용` 라고 달아주셨는데
다른 브랜치에서 이어서 작업예정이신가요?

Reply:
엇 지난번 멘토링때 말씀해주신 부분 누락했습니다. 반영했습니다!
넵 공통 Response에 Message추가하려고 했습니다.